chef_master/source/config_rb_policyfile.rst

```diff
@@ -135,6 +135,20 @@ A ``Policyfile.rb`` file may contain the following settings:
 
       named_run_list :update_app, "my_app_cookbook::default"
 
+``include_policy "NAME", *args``
+  Specify a policyfile lock to be merged with this policy. ChefDK supports pulling this lock from
+  a local file or from chef server.
+
+  ``include_policy "NAME", path: "."`` pulls the lock from ``./NAME.lock.json``.
+
+  ``include_policy "NAME", path: "./foo.lock.json"`` pulls the lock from ``./foo.lock.json``.
+
+  ``include_policy "NAME", policy_revision_id: "abcdabcdabcd", server: "http://example.com"`` pulls the policy ``NAME`` with revision id ``abcdabcdabcd`` from the ``http://example.com`` chef server.
```

Comment:
Do we fill in the chef server with the default (from config file) if it's not present or did we save that for later?

Reply:
We pitched it, never did it from what I can tell.